src/server/validateDependencies.ts

```diff
@@ -34,15 +35,49 @@ export async function validateDependencies(browserPath: string, browser: Browser
   for (const directoryPath of directoryPaths)
     lddPaths.push(...(await executablesOrSharedLibraries(directoryPath)));
   const allMissingDeps = await Promise.all(lddPaths.map(lddPath => missingFileDependencies(lddPath)));
-  const missingDeps = new Set();
+  const missingDeps: Set<string> = new Set();
   for (const deps of allMissingDeps) {
     for (const dep of deps)
       missingDeps.add(dep);
   }
   if (!missingDeps.size)
     return;
-  const deps = [...missingDeps].sort().map(dep => '   ' + dep).join('\n');
-  throw new Error('Host system is missing the following dependencies to run browser\n' + deps);
+  // Check Ubuntu version.
+  const missingPackages = new Set();
+  const osRelease = await readFileAsync('/etc/os-release', 'utf8').catch(e => '');
+  const [ubuntuName = '', ubuntuVersion = ''] = osRelease.trim().split('\n');
```

Comment:
Is the order of entries actually predefined? Not a big deal but using a map could be more reliable.

Reply:
fair enough. Done.